### DISCUSSION TITLE
what does the giveaway sniper do?

### DISCUSSION THREAD

**Discussion by Zilaeed03** [Vedza/NitroSniperGo#254] [Q&A]
Hey
I joined some few giveaways server but i didn't even win 1 giveaways does this auto react only on the gaws and there is any way i can enabke an option with which i can win the gaws? sorry im pretty new to this that's why i ask.

**Comment by DarkGD123** [comment]
when you enable the giveaway sniper, it will only sniper new giveaways not the giveaway that started hours ago

**Reply by Zilaeed03** [reply]
@DarkGD123 hey can i have your discord tag for help if u don't mind ?

**Reply by DarkGD123** [reply]
@Zilaeed03 GitYeet#2560

**Reply by Zilaeed03** [reply]
aight thanks man

**Comment by Zilaeed03** [comment]
my name is !OMY there

**Reply by Zilaeed03** [reply]
@DarkGD123 accept friend req man on discord
